src/data/models.sql

```diff
@@ -0,0 +1,296 @@
+CREATE TABLE `student` (
+	`id` varchar(10) NOT NULL,
+	`studentName` varchar(13) NOT NULL,
+	`bitsId` varchar NOT NULL UNIQUE,
```

Comment:
mysql needs a definite varchar length. This has to be changed across all the fields with no length provided